docs/docs/config/system.md

```diff
@@ -0,0 +1,96 @@
+
+If `CONFIG_ZMK_HID_REPORT_TYPE_HKRO` is enabled, it may be configured with the following options:
+
+| Config                                | Type | Description                                       | Default |
+| ------------------------------------- | ---- | ------------------------------------------------- | ------- |
+| `CONFIG_ZMK_HID_KEYBOARD_REPORT_SIZE` | int  | Number of keyboard keys simultaneously reportable | 6       |
+
+Exactly zero or one of the following options may be set to `y`. The first is used if none are set.
+
+| Config                                        | Description                                                                          |
+| --------------------------------------------- | ------------------------------------------------------------------------------------ |
+| `CONFIG_ZMK_HID_CONSUMER_REPORT_USAGES_FULL`  | Enable all consumer key codes, but may have compatibility issues with some host OSes |
+| `CONFIG_ZMK_HID_CONSUMER_REPORT_USAGES_BASIC` | Prevents using some consumer key codes, but allows compatibility with more host OSes |
+
+### USB
+
+| Config                            | Type   | Description                             | Default         |
+| --------------------------------- | ------ | --------------------------------------- | --------------- |
+| `CONFIG_USB`                      | bool   | Enable USB drivers                      |                 |
+| `CONFIG_USB_DEVICE_VID`           | int    | The vendor ID advertised to USB         | `0x1D50`        |
+| `CONFIG_USB_DEVICE_PID`           | int    | The product ID advertised to USB        | `0x615E`        |
+| `CONFIG_USB_DEVICE_MANUFACTURER`  | string | The manufacturer name advertised to USB | `"ZMK Project"` |
+| `CONFIG_USB_HID_POLL_INTERVAL_MS` | int    | USB polling interval in milliseconds    | 1               |
+| `CONFIG_ZMK_USB`                  | bool   | Enable ZMK as a USB keyboard            |                 |
+| `CONFIG_ZMK_USB_INIT_PRIORITY`    | int    | USB init priority                       | 50              |
+
+### Bluetooth
+
+See [Zephyr's Bluetooth stack architecture documentation](https://docs.zephyrproject.org/latest/guides/bluetooth/bluetooth-arch.html)
+for more information on configuring Bluetooth.
+
+| Config                                                | Type | Description                                                             | Default |
+| ----------------------------------------------------- | ---- | ----------------------------------------------------------------------- | ------- |
+| `CONFIG_BT`                                           | bool | Enable Bluetooth support                                                |         |
+| `CONFIG_BT_MAX_CONN`                                  | int  | Maximum number of simultaneous Bluetooth connections                    | 5       |
```

Comment:
Is it worth noting these two default to `6` for `ZMK_SPLIT` is enabled? We can also note in the description that includes peripheral connections for splits, if it won't be too long.